### PR TITLE
Fixed a tensorflow test which was marked as skipped.

### PR DESCRIPTION
**Pull request by zeahmed**
This PR fixes a TensorFlow test which was marked skipped. The test got obsolete because it was creating loader from string.

Also, fixes reloading of model again in the TensorFlowModel class.

### CODE REVIEW

File: test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs [hunk 1]
```diff
@@ -380,19 +380,32 @@ public void TensorFlowTransformObjectDetectionTest()
         [Fact(Skip = "Model files are not available yet")]
```
Comment (Ivanidzo4ka) [inline test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs]: >Skip = "Model files are not available yet [](start = 14, length = 41)

do you plan unskip it in separate PR?

File: test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs [hunk 2]
```diff
@@ -380,19 +380,32 @@ public void TensorFlowTransformObjectDetectionTest()
...
         public void TensorFlowTransformInceptionTest()
         {
-            var modelLocation = @"C:\models\TensorFlow\tensorflow_inception_graph.pb";
+            string inputName = "input";
+            string outputName = "softmax2_pre_activation";
+            var modelLocation = @"inception5h\tensorflow_inception_graph.pb";
```
Comment (yaeldekel) [inline test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs]: >inception5h [](start = 34, length = 11)

Has this model been uploaded to Microsoft.ML.TensorFlow.TestModels? At the time that this test was written it was skipped because we needed to handle the license issue.

**Comment by yaeldekel** [comment]
>         [Fact(Skip = "Model files are not available yet")]

This test can also be enabled, see this issue:
[https://github.com/tensorflow/models/issues/5287](https://github.com/tensorflow/models/issues/5287).

---
Refers to: test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs:339 in b5df5a4. [](commit_id = b5df5a474ac7fc36fdab2340cbdcb0a7c7ca2864, deletion_comment = False)

**Comment by yaeldekel** [comment]
>         [Fact(Skip = "Model files are not available yet")]

Also this issue #904.

---
In reply to: [469896041](https://github.com/dotnet/machinelearning/pull/2855#issuecomment-469896041) [](ancestors = 469896041)

---
Refers to: test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs:339 in b5df5a4. [](commit_id = b5df5a474ac7fc36fdab2340cbdcb0a7c7ca2864, deletion_comment = False)

File: test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs (hunk 2 above)
Comment (zeahmed) [reply test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs]: I think we haven't sorted out license issue yet. Have we?

Comment (zeahmed) [reply test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs]: Otherwise, we can download the model on the fly but it will take time during tests runs.

**Comment by zeahmed** [comment]
>         [Fact(Skip = "Model files are not available yet")]

I can do it as part of [https://github.com/dotnet/machinelearning/issues/904](https://github.com/dotnet/machinelearning/issues/904) in separate PR.

---
In reply to: [469928993](https://github.com/dotnet/machinelearning/pull/2855#issuecomment-469928993) [](ancestors = 469928993,469896041)

---
Refers to: test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs:339 in b5df5a4. [](commit_id = b5df5a474ac7fc36fdab2340cbdcb0a7c7ca2864, deletion_comment = False)

File: test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs (hunk 1 above)
Comment (zeahmed) [reply test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs]: I will do it as part of [https://github.com/dotnet/machinelearning/issues/904](https://github.com/dotnet/machinelearning/issues/904) in separate PR.

---
In reply to: [262684524](https://github.com/dotnet/machinelearning/pull/2855#discussion_r262684524) [](ancestors = 262684524)

File: test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs (hunk 2 above)
Comment (zeahmed) [reply test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs]: I will do it as part of [https://github.com/dotnet/machinelearning/issues/904](https://github.com/dotnet/machinelearning/issues/904) in separate PR.

---
In reply to: [262729625](https://github.com/dotnet/machinelearning/pull/2855#discussion_r262729625) [](ancestors = 262729625)

**Comment by zeahmed** [comment]
>         [Fact(Skip = "Model files are not available yet")]

Yael, can you please sign off if you agree?

---
In reply to: [470427764](https://github.com/dotnet/machinelearning/pull/2855#issuecomment-470427764) [](ancestors = 470427764,469928993,469896041)

---
Refers to: test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs:339 in b5df5a4. [](commit_id = b5df5a474ac7fc36fdab2340cbdcb0a7c7ca2864, deletion_comment = False)
